lib/internal/crypto/ec.js

```diff
@@ -166,7 +166,26 @@ async function ecGenerateKey(algorithm, extractable, keyUsages) {
       // Fall through
   }
   return new Promise((resolve, reject) => {
-    generateKeyPair('ec', { namedCurve }, (err, pubKey, privKey) => {
+    let genKeyType, genOpts = {};
```

Comment:
My bad, didn't test the previous suggestion.

```suggestion
    let genKeyType;
    let genOpts = {};
```